examples/olsconfig.yaml

```diff
@@ -23,7 +23,7 @@ llm_providers:
       - name: ibm/granite-13b-chat-v2
 ols_config:
   reference_content:
-#    product_docs_index_path: "./vector-db/ocp-product-docs"
+#    product_docs_index_path: "./vector_db"
```

Comment:
i think we do want product-docs in the path, so we can easily introduce additional dbs in the future in their own directories